### DISCUSSION TITLE
Show code repo branch information inside Repository View

### DISCUSSION THREAD

**Discussion by Maritn-Hammerschmid** [alefragnani/vscode-project-manager#599] [Ideas]
Hi,

Is it possible to show more information about repository inside the code repository view? (branch name, coutn of changes,...)
I found the locators from the core project package, which is part of your git-submodule ==>https://github.com/alefragnani/vscode-project-manager-core.git ,but its not visible to me, otherwise i could do a fork.

Thank you

**Comment by alefragnani** [comment]
Hi,

There is an open issue (https://github.com/alefragnani/vscode-project-manager/issues/378 ) asking for some for Git info on each project. 

Please add a comment with the the infos you would like to see, so when implemented, I could retrieve as much info as necessary.

The submodule is still closed source, so you are not able to clone.

Hope this helps

**Reply by Maritn-Hammerschmid** [reply]
Awesome, thank you, will provide more infos there
